src/cljs/main/broadfcui/common/gcs_file_preview.cljs

```diff
@@ -27,19 +27,21 @@
                         [:div {:style {:display "inline-block" :width 185}} (str label ": ")]
                         contents])
              data-empty (or (= data-size "0") (string/blank? data-size))
-             bam? (re-find #"\.ba[mi]$" (:object props))]
+             bam? (re-find #"\.ba[mi]$" (:object props))
+             img? (re-find #"\.(?:(jpe?g|png|gif|bmp))$" (:object props))
```

Comment:
Is the `?:` part needed? I guess it shouldn't hurt anything but it doesn't help readability if it doesn't need to be there.